src/app/_styles/Homepage.module.scss

```diff
@@ -20,3 +20,25 @@
     }
   }
 }
+
+.invitationCtaSection {
+  h2,
+  p {
+    color: #fff;
+  }
```

Comment:
I think we need to set this text to be larger (`1.38rem`).

Reply:
done